Assets/GothicVR/Scripts/Creator/WorldCreator.cs

```diff
@@ -42,10 +40,12 @@ public void Create(string worldName)
 
             if (FeatureFlags.I.CreateOcNpcs)
                 PxVm.CallFunction(GameData.I.VmGothicPtr, "STARTUP_OLDCAMP");
-            
-            SceneManager.MoveGameObjectToScene(worldGo, SceneManager.GetSceneByName(worldName));
+
```

Comment:
Add back the world gameobject move to scene

```suggestion
            GvrSceneManager.I.MoveToWorldScene(worldGo);
```

Reply:
this is not the proper fix :D

Reply:
Fixed bug otherwise